atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/store/WorkloadCell.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload.store;
+
+import com.google.common.primitives.Ints;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import org.immutables.value.Value;
+
+/**
+ * Cells generated by the workload service.
+ * Columns/values are expected to be random integers.
+ *
+ * Schema: Key[Integer], Column[Integer], Value[Integer].
+ */
+@Value.Immutable
+public interface WorkloadCell {
+    Integer key();
+
+    Integer column();
+
+    default Cell toCell() {
```

Comment:
I am actually a bit iffy on this -- may be be better to include this elsewhere, as this makes our workload-server-api _depend_ on atlasdb-api!